build/deps/formatters/BUILD

```diff
@@ -0,0 +1,79 @@
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+config_setting(
+    name = "linux_amd64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "linux_arm64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+config_setting(
+    name = "darwin_amd64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "darwin_arm64",
+    constraint_values = [
+        "@platforms//os:macos",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+config_setting(
+    name = "windows_amd64",
+    constraint_values = [
+        "@platforms//os:windows",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+copy_file(
+    name = "clang-format@rule",
```

Comment:
Nit: I think of "@rule" target names as an antipattern as they don't tell us anything about what the rule actually does. We can just drop that suffix for these targets and drop it in format.py and things should still work right?

Reply:
There's a bazel warning hating that the name and the output are identical.

Reply:
🤔 I thought it'd be fine, that's what we do in other cases too. Either way, it's non-blocking.